planning/scenario_planning/parking/astar_search/src/astar_search.cpp

```diff
@@ -195,18 +199,18 @@ void AstarSearch::initializeNodes(const nav_msgs::OccupancyGrid & costmap)
   // Initialize nodes
   nodes_.clear();
   nodes_.resize(height);
-  for (int i = 0; i < height; i++) {
+  for (uint32_t i = 0; i < height; i++) {
```

Comment:
Maybe I'm not understanding the change but since these are used for indexing, should these be `size_t` instead?

Reply:
There were some warnings for the comparison between `(int) i` and `(unsigned int) height`. Yes, `size_t` is also fine.